docs/product/dev-toolbar/faq.mdx

```diff
@@ -4,6 +4,55 @@ sidebar_order: 30
 description: "Frequently asked questions about the Dev Toolbar."
 ---
 
+<Expandable title="In what environments should I enable the Dev Toolbar?">
+
+Since the Dev Toolbar will be visible to users within your app, it's important to consider which environments should render it.
+
+If your web application does require authentication to access:
+- In dev and staging, always initialized the Dev Toolbar.
```

Comment:
Should we write out `development` irt env, as to not conflate with the "dev" in "Dev Toolbar"?

```suggestion
- In development and staging, always initialized the Dev Toolbar.
```